src/@commonsku/styles/AddressAutocomplete/AddressAutocompleteInput.tsx

```diff
@@ -34,20 +34,15 @@ export default function AddressAutocompleteInput({
   onInputChange = () => {},
   ...props
 }: Readonly<AddressAutocompleteInputProps>) {
-  const [currentLocation, setCurrentLocation] = useState({ lat:0, lng: 0 });
+  const [currentLocation, setCurrentLocation] = useState<LatLng | null>(null);
   const [options, setOptions] = useState<TOption[]>([]);
   const [showDropdown, setShowDropdown] = useState(options.length > 0);
-  const [sessionToken, setSessionToken] = useState<google.maps.places.AutocompleteSessionToken | undefined>(undefined);
-  const [testId] = useState(uniqueId('autocomplete-search-input-'));
 
-  useEffect(() => {
-    if (typeof window === 'undefined') { return; }
-    const places = window.google.maps.places;
-    if (!places) { return; }
-    setSessionToken(
-      new places.AutocompleteSessionToken() || undefined
-    );
-  }, []);
+  const testId = useRef(uniqueId('autocomplete-search-input-'));
```

Comment:
I think that you can probably use the `useId()` hook for this. https://react.dev/reference/react/useId.